.github/workflows/largemodel_unit_test_CI.yml

```diff
@@ -179,15 +179,16 @@ jobs:
         run: |
           if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
             export BASE_BRANCH="${{ github.event.pull_request.base.ref }}"
+            cd marqo
+            echo "Running diff-cover against branch $BASE_BRANCH"
+            git fetch origin $BASE_BRANCH:$BASE_BRANCH
+            diff-cover cov.xml --html-report diff_cov.html --markdown-report diff_cov.md \
+              --compare-branch $BASE_BRANCH
           else
-            export BASE_BRANCH=mainline
```

Comment:
Disable diff-cov on push events since the base branch is the same and there's nothing to compare. The PR process we use guarantees the coverage of new code once fail-on-threshold is set.